manifests/server.pp

```diff
@@ -29,7 +29,7 @@
 	$ips = false,	# an optional list of ip's for each in hosts[]
 	$clients = []	# list of allowed client ip's	# TODO: get from exported resources
 ) {
-	$FW = '$FW'			# make using $FW in shorewall easier
+	$fw = '$fw'			# make using $fw in shorewall easier
```

Comment:
Haven't reviewed the whole patch, but this is obviously incorrect, meaning you probably did a quick search/replace. Could you fix this and have a quick look through the whole patch? Thanks!